doc/src/build/fullnode.md

```diff
@@ -288,3 +288,98 @@ If you followed the instructions for [Building from Source](#building-from-sourc
     ```
 
 Your Full node starts on: `http://127.0.0.1:9000`.
+
+# Pruning 
+
+## Object Pruning
+
+Object pruning involves the removal of older object versions from the fullnode db. 
```

Comment:
it's always Full node unless the specific database is named "fullnode".

Spell out database